ui/lib/core/addon/templates/components/masked-input.hbs

```diff
@@ -1,7 +1,7 @@
 <div
   class="masked-input {{if this.displayOnly 'display-only'}} {{if this.allowCopy 'allow-copy'}}"
-  data-test-masked-input
-  data-test-field
+  data-test-masked-input={{this.name}}
```

Comment:
good call. does it affect other tests however?

Reply:
:disappointed: yes. It's the reason behind the test failures so I reverted these changes. I'm surprised since doing `document.querySelector('some-attribute')` when the attribute has a value will still return the node.